src/ducks/account.ts

```diff
@@ -38,18 +40,26 @@ interface InitialState {
   data: Types.AccountDetails | null;
   error?: string;
   loading: boolean;
+  // TODO - "any" ?
+  pastTransactions: Array<any>;
```

Comment:
I'm using "any" in a couple places for complex types which I'm not sure how to handle right now. Thinking of noting for now, and then putting types in a separate folder (config?) that we import later once we have a better grasp of the types we're using

Reply:
I think a good way to do this is to do something like `type Transaction any` so it can be swapped out more easily later